mongo/datadog_checks/mongo/dbm/utils.py

```diff
@@ -39,6 +39,18 @@
     ]
 )
 
+UNEXPLAINABLE_COMMANDS = frozenset(
+    [
+        "getMore",
+        "insert",
+        "update",
+        "delete",
```

Comment:
is `update set ... where col1=1` explainable?

Reply:
no. the sampled command only has the filter part of the operation but missing the update part. trying to explain such command will result in an operation error